frontend/src/views/Apps/AppsMain/AppsMain.scss

```diff
@@ -76,6 +69,7 @@
         }
 
         .body {
+          padding: 20px;
           overflow-x: hidden;
         }
       }
```

Comment:
In the nodes branch, appsMain and nodesMain share the same css file, when is merged, will have to update the changes on that file, although they're minimal